sys/kern/dtb.c

```diff
@@ -0,0 +1,34 @@
+#include <sys/klog.h>
+#include <sys/mimiker.h>
+#include <sys/dtb.h>
+#include <sys/fdt.h>
+#include <sys/kmem.h>
+#include <sys/vm.h>
+
```

Comment:
I like this file!
I can see what it does without commments

Reply:
So what is a problem?